packages/nx/src/command-line/release/version.ts

```diff
@@ -534,7 +534,7 @@ function printAndFlushChanges(tree: Tree, isDryRun: boolean) {
           isDryRun ? chalk.keyword('orange')(' [dry-run]') : ''
         }`
       );
-      printDiff('', f.content?.toString() || '');
+      printDiff('', f.content?.toString() ?? '');
     } else if (f.type === 'UPDATE') {
```

Comment:
Type check

Reply:
release version.ts is not relevant to a PR regarding bun, please revert